packages/connector/src/types.ts

```diff
@@ -67,6 +69,16 @@ export type SaveCredentialsRequestParams = {
   type: string;
 };
 
+export type RemoveCredentialsRequestParams =
+  | {
+      ids: string[];
+      claimIds: never;
```

Comment:
What is the use case for deleting credentials by claim ID?

Reply:
described in readme

Reply:
IMO still not clear

Reply:
In what use case I want to remove the credential but don't have it's ID?

Reply:
VC ID is a concatenation of issuer api url and claimId.
Usually, backend can't access to user's vc, but it could has claimId.

So, we can be sure, that claimId will be in more cases, than just id

Reply:
It's user who remove the VCs, not some backend. And user has the list of all credentials